src/dockerValidator.ts

```diff
@@ -1153,7 +1153,7 @@ export class Validator {
         "invalidDestination": "When using ${0} with more than one source file, the destination must be a directory and end with a / or a \\",
 
         "syntaxMissingEquals": "Syntax error - can't find = in \"${0}\". Must be of the form: name=value",
-        "syntaxMissingNames": "${0} names can not be blank",
+        "syntaxMissingNames": "${0} names cannot be blank",
```

Comment:
This change is how I came across this repo, it's a slightly different 🦆 from everything else.

Reply:
@jsoref Thank you for your contribution!